esmvaltool/references/duemenil00mpimr.bibtex

```diff
@@ -0,0 +1,3 @@
+@article{duemenil00mpimr,
+    title = "Duemenil Gates et al., Observed historical discharge data from major rivers for climate model validation. Max Planck Institute for Meteorology Report 307, Hamburg, Germany, 2000."
```

Comment:
This should be split in the respective fields (author, title, etc).

Reply:
done.